model/riscv_pmp_regs.sail

```diff
@@ -43,6 +43,15 @@ bitfield Pmpcfg_ent : bits(8) = {
 register pmpcfg_n : vector(64, Pmpcfg_ent)
 register pmpaddr_n : vector(64, xlenbits)
 
+bitfield Mseccfg_ent : xlenbits = {
+  RLB  : 2,       /* Rule Locking Bypass */
+  MMWP : 1,       /* Machine Mode Whitelist Policy */
+  MML  : 0        /* Machine Mode Lockdown */
+}
+
+register mseccfg  : Mseccfg_ent
+register mseccfgh : bits(32)
```

Comment:
Normally we don't treat this as a separate register, we just make `bitfield Mseccfg : bits(64)` and then read/write the appropriate bits of it.

We have an implementation of `mseccfg` like that internally; I'll make a PR for it.